staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/compatibility.go

```diff
@@ -231,8 +231,8 @@ func CompatibilityTestFuzzer(scheme *runtime.Scheme, fuzzFuncs []interface{}) *f
 		func(f *[]metav1.ManagedFieldsEntry, c fuzz.Continue) {
 			field := metav1.ManagedFieldsEntry{}
 			c.Fuzz(&field)
-			if field.Fields != nil {
-				field.Fields.Raw = []byte("{}")
+			field.Fields = &metav1.Fields{
+				Raw: []byte(`{"18":{"19":null}}`),
```

Comment:
is there a way to fuzz or populate the corresponding structured-merge-diff struct and marshal it to produce this?

Reply:
Yes, there's this: https://github.com/kubernetes-sigs/structured-merge-diff/blob/0c1d754dd6485aa156ff3c8c599079adac6ec82f/fieldpath/set_test.go#L30

Reply:
It's not deterministic but we could make a deterministic version

Reply:
a small deterministic version sounds good